### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by specfazhou**

### CODE REVIEW

File: _summaries/chen2020a_1.md [hunk 1]
```diff
@@ -1,19 +1,40 @@
...
-score: # How did you like this paper 0(dislike) to 10(love)
+author: specfazhou
+score: 8/10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
+* What is the core idea? <br/>
+This paper comes up with a simple framework for contrasive learning of visual representation, which not only archieves state of the art accuracy, but also has simplier structure. The result of SimCLR is shown in the following picture <br/>
```
Comment (rll2396) [inline _summaries/chen2020a_1.md:12]: Since you bring up that this framework is simpler, I think it could be good to mention somewhere what complicated things previous works were doing.

File: _summaries/chen2020a_1.md [hunk 2]
```diff
@@ -1,19 +1,40 @@
...
+<img src = "chen2020_a_3.png">
+
+* How is it realized (technically)? <br/>
+The architecture of the SimCLR is shown as the following picture <br/>
+<img src = "chen2020_a_1.png">
+As you can see from the picture. First, for each data sample, x, the model generates two views of of the data sample by sequentially applying three data augmentation methods -- random cropping, random color distortions, and random Gauessian blur. Then the model applies encoder $$f(-)$$. There are various choices of the encoder, but in this paper, the authors use ResNet. Projection head $$g(-)$$ is MLP with one hidden layer, and it uses ReLU as activation function. The reason we need projection head $$g(-)$$ is that it maps representations to the space where contrasive loss can be applied. <br/>
+
+The training process is shown as following. <br/>
+<img src = "chen2020_a_2.png">
+
```
Comment (rll2396) [inline _summaries/chen2020a_1.md:22]: Elaborate a bit on the contrastive loss. What inputs should be "similar" in the latent space, and which ones shouldn't?

File: _summaries/chen2020a_1.md [hunk 3]
```diff
@@ -1,19 +1,40 @@
...
+
 * How well does the paper perform?
-* What interesting variants are explored?
+1. Larger batch size has more advantage than smaller batch size when the number of training epochs is small, but this advantage disappears when the number of epochs increases, as shown in the following picture <br/>
```
Comment (rll2396) [inline _summaries/chen2020a_1.md:25]: A major point that the authors highlight throughout their ablation studies is that contrastive learning benefits more that supervised methods, so include that.

File: _summaries/chen2020a_1.md (hunk 2 above)
Comment (rll2396) [reply _summaries/chen2020a_1.md:22]: Could be worth mentioning the optimizer they use since it's not the typical ADAM/SGD. Also, how long does their network take to train?

**Review by rll2396 — COMMENTED** [review]
Many typos, "contrasive" stands out
Other than that, good review.

File: _summaries/chen2020a_1.md [hunk 4]
```diff
@@ -1,19 +1,40 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {chen2020a_1}
```
[the remark this answers is not in the document]
Comment (specfazhou) [reply _summaries/chen2020a_1.md:4]: deleted, thanks

File: _summaries/chen2020a_1.md [hunk 5]
```diff
@@ -1,19 +1,40 @@
...
+<img src = "chen2020_a_7.png">
+5. Choice of projection heads is very important<br/>
+<img src = "chen2020_a_8.png">
+6. SimCLR achieves state of the art results on ImageNet classification
+<img src = "chen2020_a_10.png">
+8. SimCLR achieves competitive results on 12 natural image classification datasets by linear evaluation and fine tuning. 
+<img src = "chen2020_a_9.png">
+
 
-## TL;DR
```
[the remark this answers is not in the document]
Comment (specfazhou) [reply _summaries/chen2020a_1.md:16]: added, thanks

File: _summaries/chen2020a_1.md (hunk 1 above)
Comment (specfazhou) [reply _summaries/chen2020a_1.md:12]: I think during the presentation, other students will say previous works first, maybe I just leave it like this

Comment (specfazhou) [reply _summaries/chen2020a_1.md:12]: added the fact that previous works require memory bank and specialized architectures

File: _summaries/chen2020a_1.md (hunk 2 above)
Comment (specfazhou) [reply _summaries/chen2020a_1.md:22]: I see, thanks, that's very inspiring.

File: _summaries/chen2020a_1.md (hunk 3 above)
Comment (specfazhou) [reply _summaries/chen2020a_1.md]: added, thanks

**Review by zhaoyue-zephyrus — APPROVED** [review]
